src/main/java/pl/akademia/api/OrderRestController.java

```diff
@@ -0,0 +1,59 @@
+package pl.akademia.api;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import pl.akademia.api.model.Order;
+
+import java.sql.Date;
+import java.util.List;
+
+@RestController
+@RequestMapping("/api/v1")
+public class OrderRestController {
+
+    private final OrderService orderService;
+
+    public OrderRestController(OrderService orderService) {
+        this.orderService = orderService;
+    }
+
+    @GetMapping("/orders")
+    public ResponseEntity<List<Order>> getAllOrders(){
```

Comment:
getAllOrders(@RequestParam, @Requestparam(required = false))

Reply:
to mozesz nizej sprawdzic czy min oraz max == null, wowczas metoda dziala bez zmian, jak nie null to pobierasz z innej metody seriwsu